DelvCD/Helpers/ClipRectsHelper.cs

```diff
@@ -162,24 +168,35 @@ public unsafe void Update()
                         continue;
                     }
 
-                    string? name = addon->NameString;
-                    if (name == null || !AddonNames.Contains(name))
```

Comment:
Removing this condition doens't seem right

Reply:
This is how we did it for DelvUI. We used to traverse "internal static List<string> AddonNames = new List<string>()" and removing it that way but then we moved away from that, I did forget to remove the List tho, doing it that way will require us adding every single addon name we want to clip (and thus is maintenance heavy)

Reply:
Then we should remove the old list